pom.xml

```diff
@@ -1414,8 +1414,8 @@
     <powermockVersion>1.6.4</powermockVersion>
     <okhttpVersion>3.10.0</okhttpVersion>
     <okioVersion>1.14.0</okioVersion>
-    <opennmsApiVersion>0.4.1</opennmsApiVersion>
-    <opennmsApiVersionOsgi>0.4.1</opennmsApiVersionOsgi>
+    <opennmsApiVersion>0.4.2-SNAPSHOT</opennmsApiVersion>
```

Comment:
Currently, I have no clue how the release cycle for the integration layer works. Do we have to change this before we merge to `release-26` but after we released the integration API?

Reply:
Me neither. Every new commit in the master of the Integration API will release a new snapshot build automatically. So OpenNMS will compile and work. But of course we should reference a fixed version of the Integration Api before we release a new Opennms version.

@RangerRick would you know how we should proceed here?

Reply:
Talked to @RangerRick : It is ok to merge now. We just need to make a new Integration API version before a release. He will have his release script check for that :-)

Reply:
yup, updated... https://github.com/OpenNMS/opennms-repo/commit/ee4b7b6d971a3e4d20bc683c862be55bdb0f6bac